packages/react-components/src/components/Alert/Alert.tsx

```diff
@@ -55,50 +70,77 @@ const baseClass = 'alert';
 export const Alert: React.FC<React.PropsWithChildren<AlertProps>> = ({
   children,
   className,
+  primaryButton,
+  secondaryButton,
   kind = 'info',
   onClose,
   ...props
 }) => {
   const containerRef = React.useRef<HTMLDivElement>(null);
-  const [isSmallContainer, setIsSmallContainer] = React.useState(false);
+  const [containerSize, setContainerSize] = React.useState<string | false>(
+    false
+  );
```

Comment:
What is the reason to mix string and false type here? Can we use null instead of false?